assets/css/woocommerce.scss

```diff
@@ -48,6 +48,10 @@ p.demo_store {
 	clear: both;
 }
 
+option[hidden] {
+	display: none;
```

Comment:
I was hoping `display:none` would work as a fallback but it doesn't seem to be supported by browsers that ignore the `hidden` attr.